src/content/mod_configuration.cpp

```diff
@@ -237,10 +238,24 @@ void ModConfiguration::resolveDependencies()
 				mod.unsatisfied_depends.insert(optdep);
 		}
 		// if a mod has no depends it is initially satisfied
-		if (mod.unsatisfied_depends.empty())
-			satisfied.push_back(mod);
-		else
-			unsatisfied.push_back(mod);
+		if (mod.unsatisfied_depends.empty()) {
```

Comment:
It looks like the formatting is a bit messy here

Reply:
You ar right. Looks like spaces are used here as indent by mistake.